syz-manager/manager.go

```diff
@@ -182,12 +184,18 @@ func RunManager(cfg *mgrconfig.Config, target *prog.Target, sysTarget *targets.T
 		reproRequest:          make(chan chan map[string]bool),
 		usedFiles:             make(map[string]time.Time),
 		saturatedCalls:        make(map[string]bool),
+		kcovFilter:            &CoverFilter{enableFilter: false},
```

Comment:
It would be much more natural to use `mgr.kcovFilter == nil` to denote that it's not enabled and remove the enableFilter member.